### ISSUE TITLE
[BUG] UWP Grouped ListView Scroll Bug on Windows 1803 and later

### ISSUE THREAD

**Issue by DerGary** [microsoft/microsoft-ui-xaml#739]
Hello,

I am experiencing a strange bug on all of my UWP Apps where all ScrollViewers in the App break and I need to restart the app. I already explained this on the developer community and they redirected me here. The original thread: https://developercommunity.visualstudio.com/content/problem/569414/bug-uwp-grouped-listview-scroll-bug-on-windows-180.html

## Short Description:

After Navigating a few times all ScrollViewers inside ListView and GridView stop working when the Mouse Wheel is used for Scrolling. The ScrollViewer does scroll but the Page does not get rendered anymore. (Scrolling with mouse courser does work)

## Prerequisits:

Sample uwp project:
[SampleApp.zip](https://github.com/microsoft/microsoft-ui-xaml/files/3205833/SampleApp.zip)

1. A Windows PC with at least build 17134 or 17763 (I reproduced the error on both versions) (the error does not occur on 14393 and on 15063) (I didn't test other versions)
2. A UWP Project (I don't know if it makes a difference if I target different UWP version. I have the bug both targeting 10240 and 15063 and 17134)
3. Two Pages where at least one of them has a grouped listview or grouped gridview
4. Navigating forth and back on these two pages while scrolling with the mouse wheel until the ScrollViewer breaks.

## Long Description:

After Starting the App I Navigated forth and back on the two pages while scrolling. Randomly the View rendering gets slower and sometimes sooner sometimes later (sometimes I had to restart the App 1 or 2 times and then I got the error pretty soon) the ScrollViewer inside the ListView or GridView break. After that when the Mouse Wheel is used to scroll the ScrollIndicator does follow the Scrolling of the Mouse Wheel but the Content of the Page does not follow the Scrolling. Strangly the Mouse Pointer Event does seem to be where it should be because you can see that not the row where the mouse pointer is gets highlighted but the row where the mouse pointer should be if the content would be scrolling to where it should be. After The ScrollViewer is broken all other ScrollViewers in the App are broken even on different pages after a navigation nothing scrolls as it is supposed to. There is no way to fix it at this stage apart from closing and restarting the entire app.

I captured a video of my screen while I reproduced the Issue and added it to the attachments as well. In this video I am scrolling very rapidly but I have also reproduced this with normal use.

**Expected behavior**
The ScrollViewer does not break and scrolls normally

**Screenshots**
[Reproduction Video.zip](https://github.com/microsoft/microsoft-ui-xaml/files/3205832/Reproduction.Video.zip)

**NuGet package version:**
I don't have a nuget package with the name Microsoft.UI.Xaml I only have Microsoft.NETCore.UniversalWindowsPlatform 6.2.8

## Which Windows versions did you see the issue on? Leave blank if you didn't try that version. 
| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2019 Update (18362)            |  |
| October 2018 Update (17763)        | Yes   |
| April 2018 Update (17134)          | Yes  |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            | No   |
| Anniversary Update (14393)         | No  |

## Which device form factors did you see the issue on? Leave blank if you didn't try that device. 

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes   |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Comment by micahl** [comment]
@DerGary thanks for reporting this.  Love having videos / pics to show a repro + detailed steps.  This sounds very similar to issues we've seen in the past that have been very hard to track down.  We believe we identified and fixed one instance of this, but maybe not all occurences (see #666).  Could you give it a try on a recent build?

**Comment by DerGary** [comment]
@micahl thank you for your fast answer. Sorry I opened another issue. On my initial search I didn't found an open issue. 

I just tried to reproduce the issue on 1903 in a Virtual Machine over a remote desktop connection and I could not reproduce it. (But I don't know if it worked because of the fact that it is a virtual machine or the remote desktop connection changes the scroll input signal in any way?)

**Comment by micahl** [comment]
No problem.  I'd rather see duplicate issues opened than having a potentially unique issue remain unknown.

Based on your repro video I believe this issue is the same as the one we fixed which could be triggered by switching between different viewports.  Wim [re-posted what I shared with him](https://github.com/microsoft/microsoft-ui-xaml/issues/666#issuecomment-492383456) when we investigated the root cause.
